code/FsComposite/Composite.Test/App.config

```diff
@@ -1,6 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
+
 <configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7" />
-    </startup>
+  <appSettings>
+    <add key="GithubConfigPath" value="..\..\DataSourceConfigs\github_config.json" />

```

Comment:
this one is interesting. where does it come from?

Reply:
In the ProcessSample project there is same setting in the app.config file.